src/components/admin/product/ProductCard.tsx

```diff
@@ -15,97 +15,93 @@ const unSellableStyle = `
-  width: 220px;
-  height: 320px;
-  padding: 25px 37px;
-  border-radius: 13px;
+  width: 180px;
+  height: 228px;
+  padding: 12px;
+  border-radius: 16px;
+  border: 1px solid #e8eef2;
   box-sizing: border-box;
   ${(props) => (props.isSellable ? sellableStyle : unSellableStyle)}
-  ${colFlex({ justify: 'center', align: 'center' })}
-`;
-
-const SubContainer = styled.div`
-  width: 100%;
-  height: 100%;
-  gap: 15px;
-  ${colFlex({ justify: 'center', align: 'center' })};
+  ${colFlex({ justify: 'space-between', align: 'center' })}
 `;
 
 const Title = styled.div`
-  font-size: 20px;
+  line-height: 24px;
+  font-size: 16px;
   font-weight: 700;
 `;
 
 const Price = styled.div`
-  font-size: 20px;
+  line-height: 20px;
+  font-size: 12px;
   font-weight: 500;
 `;
 
 const ContentContainer = styled.div`
   cursor: pointer;
-  gap: 15px;
+  width: 100%;
+  flex: 1;
   ${colFlex({ justify: 'center', align: 'center' })};
 `;
 
 const TextContainer = styled.div<{ isSellable: boolean | null }>`
-  height: 80px;
-  color: ${(props) => (props.isSellable ? Color.BLACK : '#B2B2B2')};
+  color: ${(props) => (props.isSellable ? '#464a4d' : '#B2B2B2')};
+  padding-bottom: 12px;
   ${colFlex({ justify: 'center', align: 'center' })}
 `;
 
-const ImageContainer = styled.div<{ isSellable: boolean | null }>`
-  width: 150px;
-  height: 120px;
-  filter: ${(props) => (props.isSellable ? 'none' : 'grayScale(50%)')};
-  ${rowFlex({ justify: 'center', align: 'center' })}
-`;
-
 const Image = styled.img<{ isSellable: boolean | null }>`
-  width: 125px;
-  height: 125px;
-  border-radius: 5px;
-  border: 0.5px solid #e8e8e8;
-  opacity: ${(props) => (props.isSellable ? 1 : 0.2)};
+  width: 100px;
+  height: 100px;
+  border-radius: 6px;
+  border: 1px solid #e8eef2;
+  opacity: ${(props) => (props.isSellable ? 1 : 0.4)};
 `;
 
-const SwitchButtonContainer = styled.div`
+const SelectorWrapper = styled.div`
   width: 100%;
-  height: 40px;
-  ${rowFlex({ justify: 'center', align: 'center' })}
+  ${colFlex({ justify: 'center', align: 'center' })}
 `;
 
-interface Props {
+interface ProductCardProps {
   product: Product;
   onClick?: () => void;
 }
 
-function ProductCard({ product, onClick }: Props) {
+function ProductCard({ product, onClick }: ProductCardProps) {
   const { workspaceId } = useParams<{ workspaceId: string }>();
   const { editProductSellable } = useAdminProducts(workspaceId);
 
+  const getCurrentStatus = (): ProductStatus => {
+    if (product.isSellable) return 'SELLING';
+    return 'HIDDEN';
+  };
+
+  const handleStatusChange = (newStatus: ProductStatus) => {
+    if (newStatus === 'SELLING') {
+      editProductSellable(product.id, true);
+    } else if (newStatus === 'HIDDEN') {
+      editProductSellable(product.id, false);
+    } else if (newStatus === 'SOLD_OUT') {
+      // TODO: 품절 처리 로직 (product에 status 추가)
+      // 현재는 판매 불가 상태로 처리
+      editProductSellable(product.id, false);
+    }
```

Comment:
```suggestion
    if (newStatus === 'SELLING') {
      editProductSellable(product.id, true);
    } else {
      editProductSellable(product.id, false);
    }
```

이렇게 적고 todo 로 삭제되어야 하는 부분이라고 적으시죠!
차피 나중에는 editProductStatus(product.id, newStatus)와 같이 적기만 할테니깐요~